apps/01PigLatin.js

```diff
@@ -3,49 +3,83 @@
 var assert = require('assert');
 var readline = require('readline');
 var rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
+    input: process.stdin,
+    output: process.stdout
 });
 
 
 function pigLatin(word) {
 
-  // Your code here
+    var firstLetter = word[0];
+
+    word = word.toLowerCase();
+
+    var vowelIndex = -1;
+
+    if ((word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex) || vowelIndex === -1) {
+        vowelIndex = word.indexOf('a');
+    }
+
+    if ((word.indexOf('e') > -1 && word.indexOf('e') < vowelIndex) || vowelIndex === -1) {
+        vowelIndex = word.indexOf('e');
+    }
+
+    if ((word.indexOf('i') > -1 && word.indexOf('i') < vowelIndex) || vowelIndex === -1) {
+        vowelIndex = word.indexOf('i');
+    }
+    if ((word.indexOf('o') > -1 && word.indexOf('o') < vowelIndex) || vowelIndex === -1) {
+        vowelIndex = word.indexOf('o');
+    }
+
+    if ((word.indexOf('u') > -1 && word.indexOf('u') < vowelIndex) || vowelIndex === -1) {
+        vowelIndex = word.indexOf('u');
+    }
+
+    if ((word.indexOf('y') > -1 && word.indexOf('y') < vowelIndex) || vowelIndex === -1) {
+        vowelIndex = word.indexOf('y');
+    }
+    var firstPart = word.slice(0, vowelIndex);
+    var restWord = word.slice(vowelIndex, word.length);
+
+    if (vowelIndex === 0) {
+        return word + 'yay';
+    } else {
+        return restWord + firstPart + 'ay';
+    }
 
 }
 
 
 function getPrompt() {
-  rl.question('word ', (answer) => {
-    console.log( pigLatin(answer) );
-    getPrompt();
-  });
+    rl.question('word ', (answer) => {
```

Comment:
For your linting indentation issues, if you are using Visual Studio Code, and you look towards the bottom-right of the application, you will find something saying 'Spaces: 2' or 'Tabs: 2'. If you click on that and click on 'Indent Using Spaces' or 'Indent Using Tabs', you can choose how many to use in your document. In addition, you can convert the entire file's indentation to spaces or tabs.

Good luck, and ask me if you have any other questions.